src/easy_config/__init__.py

```diff
@@ -165,6 +168,10 @@ def load(
         if _additional_files:
             for f in _additional_files:
                 values.update(cls._load_file(f))
+        if _lookup_config_envvar is not None:
+            envvar = f'{cls.NAME.upper()}_{_lookup_config_envvar.upper()}'
+            f = os.environ[envvar]
```

Comment:
What happens if the envvar is unset?

Reply:
muahaha explosions. Should we make a custom wrapper?

Reply:
Nah, just use `.get` and check for None or whatever or put a try around it.

Add a test.